lib/routes/account.js

```diff
@@ -213,7 +213,7 @@ module.exports = function (
               emailCode: account.emailCode,
               emailVerified: account.emailVerified,
               verifierSetAt: account.verifierSetAt,
-              createdAt: optionallyOverrideCreatedAt(),
+              createdAt: parseInt(query._createdAt),
```

Comment:
Okay I think once I wrote that comment above things make a bit more sense and the main the source of confusion that remains is probably the `this.accountCreatedAt = NaN`